x/oracle/keeper.go

```diff
@@ -0,0 +1,364 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	gogotypes "github.com/gogo/protobuf/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// Keeper of the oracle store
+type Keeper struct {
+	cdc        codec.BinaryCodec
+	storeKey   sdk.StoreKey
+	paramSpace paramstypes.Subspace
+
+	accountKeeper types.AccountKeeper
+	bankKeeper    types.BankKeeper
+	distrKeeper   types.DistributionKeeper
+	StakingKeeper types.StakingKeeper
+
+	distrName string
+}
+
+// NewKeeper constructs a new keeper for oracle
+func NewKeeper(cdc codec.BinaryCodec, storeKey sdk.StoreKey,
+	paramspace paramstypes.Subspace, accountKeeper types.AccountKeeper,
+	bankKeeper types.BankKeeper, distrKeeper types.DistributionKeeper,
+	stakingKeeper types.StakingKeeper, distrName string) Keeper {
```

Comment:
nit: long functions like this should be broken down as:

```
func(
  arg1,
  arg2,
  ...,
) Keeper {
  // ...
}
```

```suggestion
func NewKeeper(cdc codec.BinaryCodec, storeKey sdk.StoreKey,
	paramspace paramstypes.Subspace, accountKeeper types.AccountKeeper,
	bankKeeper types.BankKeeper, distrKeeper types.DistributionKeeper,
	stakingKeeper types.StakingKeeper, distrName string) Keeper {
```